data/858/983/01/85898301.geojson

```diff
@@ -30,6 +30,9 @@
     "name:deu_x_variant":[
         "Grossfeldsiedlung"
     ],
+    "name:ger_x_preferred_x_variant":[
```

Comment:
This should be `deu`, looks like we'll need to remap it from the input?

Reply:
Please audit the language codes used in the SHP's DBF and make sure there's not any others we should remap.

Reply:
Ill update in a new commit.